src/views/splash/l10n.json

```diff
@@ -38,14 +38,16 @@
     "teacherbanner.classesButton": "My Classes",
     "teacherbanner.faqButton": "Teacher Account FAQ",
 
-    "hocbanner.title": "Get Creative with Coding!",
+    "hocbanner.title": "Tell Your Story with Scratch!",
```

Comment:
```suggestion
    "hocbanner.title": "Get Creative with Coding!",
    "hocbanner.titleTellStory": "Tell Your Story with Scratch!",
```

Reply:
And then src/views/splash/hoc/top-banner.jsx line 18 could change from `hocbanner.title` to `hocbanner.titleTellStory`

Reply:
I think you still need to change src/views/splash/hoc/top-banner.jsx line 18 to use `hocbanner.titleTellStory`?